### PR TITLE
FIX: send activation email if user have unconfirmed email address

### PR DESCRIPTION
**Pull request by techAPJ**
@tgxworld can you review this PR? Thanks!

### CODE REVIEW

**Comment by techAPJ** [comment]
Build failure is not related.

File: spec/controllers/users_controller_spec.rb [hunk 1]
```diff
@@ -1440,6 +1442,20 @@ def post_user
         end
       end
 
+      context 'for an activated account with unconfirmed email' do
+        it 'should send an email' do
+          unconfirmed_email_user = Fabricate(:user, active: true)
+          email_token = unconfirmed_email_user.email_tokens.create(email: unconfirmed_email_user.email).token
```
Comment (tgxworld) [inline spec/controllers/users_controller_spec.rb]: I don't think we need this variable?

**Comment by tgxworld** [comment]
LGTM

File: spec/controllers/users_controller_spec.rb (hunk 1 above)
Comment (techAPJ) [inline spec/controllers/users_controller_spec.rb]: Oh yeah, that's correct. I just want to create an email token here. Removed the variable.

**Pull request merged**